packages/frontend/src/components/NavBar/HelpMenu/index.tsx

```diff
@@ -0,0 +1,105 @@
+import {
+    Button,
+    Icon,
+    PopoverInteractionKind,
+    Position,
+    Spinner,
+} from '@blueprintjs/core';
+import { Popover2 } from '@blueprintjs/popover2';
+import React, { FC } from 'react';
+import { SpinnerWrapper } from '../NavBar.styles';
+import {
+    ButtonWrapper,
+    HelpItem,
+    IconContainer,
+    ItemCTA,
+    ItemDescription,
+    LinkWrapper,
+    MenuWrapper,
+} from './HelpMenu.styles';
+
+interface Props {
+    projectId: string | undefined;
+}
+
+const HelpMenu: FC<Props> = ({ projectId }) => {
+    const openChatWindow = () => {
+        (window as any).$chatwoot?.toggle('true');
+    };
+
+    return (
+        <>
+            <Popover2
+                interactionKind={PopoverInteractionKind.CLICK}
+                content={
+                    !projectId ? (
```

Comment:
None of these menu options seem to be dependent on the project id. If so, so we can remove this.